docs/security.md

```diff
@@ -74,6 +74,32 @@ The snippet below shows the how the credentials can be specified in the configur
 
 The property `dt.authentication.principal` specifies the Kerberos user principal and `dt.authentication.keytab` specifies the absolute path to the keytab file for the user.
 
+### Webservices security
+
+Alongside every Apex application is an application master process running called Streaming Container Manager (STRAM). STRAM manages the application by handling the various control aspects of the application such as orchestrating the execution of the application on the cluster, playing a key role in scalability and fault tolerance, providing application insight by collecting statistics among other functionality.
+
+STRAM provides a web service interface to introspect the state of the application, its various components and to make dynamic changes to the applications. Some examples of supported functionality are getting resource usage and partition information of various operators, getting operator statistics and changing properties of running operators.
```

Comment:
"STRAM provides a web service interface to introspect the state of the application and its components, and also to make dynamic changes to the applications."